examples/k8s.yaml

```diff
@@ -116,7 +116,7 @@ provision:
     EOF
     kubeadm init --config kubeadm-config.yaml
     # Installing a Pod network add-on
-    kubectl apply -f https://raw.githubusercontent.com/flannel-io/flannel/v0.14.0/Documentation/kube-flannel.yml
+    kubectl apply -f https://raw.githubusercontent.com/flannel-io/flannel/master/Documentation/kube-flannel.yml
     # Control plane node isolation
```

Comment:
Can we specify the commit hash?

Reply:
Unfortunately the tagging is broken again, but we can find the commi from when the manifest was bumped...